packages/block-library/src/post-title/block.json

```diff
@@ -23,13 +23,9 @@
 		"__experimentalFontSize": true,
 		"__experimentalLineHeight": true,
 		"__experimentalSelector": {
-			"core/post-title/h1": "h1",
```

Comment:
It seems we have an issue both post-title and heading block map to the same CSS selectors so CSS output from one block will overwrite the CSS output of the other blocks. One post title the selector should probably also include a class.